crates/client/src/client/schema.rs

```diff
@@ -328,3 +328,9 @@ impl From<TryFromSliceError> for ConversionError {
         ConversionError::BytesLength
     }
 }
+
+#[derive(cynic::QueryFragment, Clone, Debug, Eq, PartialEq)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub struct Version {
```

Comment:
I moved this out of `chain.rs` and into the parent level `schema.rs` for reuse in other submodules.